freqtrade/commands/cli_options.py

```diff
@@ -413,6 +413,11 @@ def __init__(self, *args, **kwargs):
         metavar='INT',
         default=750,
     ),
+    "skip_trades": Arg(
+        '--skip-trades',
+        help='Skip using trades file from backtesting and DB.',
+        action='store_true',
+    ),
```

Comment:
Personally, i would prefer to use `--no-trades`. 

While technically the trades loading is skipped, no-trades seems more intuitive to me - but it's a personal preference - so if you prefer skip-trades that's fine, too

Reply:
changes to --no-trades